MMM-EmbedYoutube.js

```diff
@@ -41,6 +42,7 @@ Module.register("MMM-EmbedYoutube", {
 			}
 		}
 		params += (this.config.autoplay) ? "autoplay=1" : "autoplay=0";
+		params += (this.config.mute) ? "&mute=1" : "&mute=0";
```

Comment:
Where did you get `mute`? I can't find this parameters from Youtube IFrame API docs.